src/pages/home.css

```diff
@@ -0,0 +1,205 @@
+#page {
+	margin-left: 40px;
+	margin-right: 40px;
+	display: block;
+}
+
+/* #region Page Header */
+
+#page-header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+}
+
+#page-header-main {
+	display: flex;
+	align-items: center;
+}
+
+#page-header-main-generic-profile-icon {
+	width: 40px;
+	height: 40px;
+	background-color: orange;
+	border-radius: 50px;
+}
+
+#page-header-main-text {
+	width: 100%;
+	padding-left: 10px;
+
+	font-family: 'Inter';
```

Comment:
On my device, even though you have the font family specified, I'm still not seeing the right font.

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/55410292/187334157-3b2bcece-22e4-48ef-9e1a-0278e4ba0acb.png">

Reply:
Glad you caught that! It turns out that we weren't actually setting the specific font. I didn't even realize that all of the text was different than the prototype lol.